model-engine/requirements.in

```diff
@@ -14,7 +14,7 @@ cloudpickle==2.1.0
 croniter==1.4.1
 dataclasses-json>=0.5.7
 datadog-api-client==2.11.0
-datadog~=0.46.0
+git+https://github.com/DataDog/datadogpy.git@a764cb054d9a3089a3a23779be8d64f18346aeb9
```

Comment:
I thought @louisbarrett wanted us in these situations to use our own fork? Or is this ok?

Reply:
this PR is not merged though, we're using our fork of datadogpy for this right now